gtkmm.json

```diff
@@ -45,9 +45,9 @@
             "name": "sigc++",
             "sources": [
                 {
-                    "sha256": "dda176dc4681bda9d5a2ac1bc55273bdd381662b7a6d49e918267d13e8774e1b",
+                    "sha256": "c3d23b37dfd6e39f2e09f091b77b1541fbfa17c4f0b6bf5c89baef7229080e17",
                     "type": "archive",
-                    "url": "https://ftp.gnome.org/pub/GNOME/sources/libsigc++/2.10/libsigc++-2.10.6.tar.xz"
+                    "url": "https://download.gnome.org/sources/libsigc++/3.6/libsigc++-3.6.0.tar.xz"
```

Comment:
```
../meson.build:106:13: ERROR: Dependency "sigc++-2.0" not found, tried pkgconfig and cmake
```

I mean the local build would fail the same way.

Reply:
Man is the only animal that trips twice over the same stone! 🙂

Reply:
… on the same stone man put in front to trip on.